src/rest-server/yarn.lock

```diff
@@ -5,6 +5,7 @@
 "@octokit/rest@~15.9.5":
```

Comment:
what's the purpose of this file, as not seen it be used by which file

Reply:
the new integrity fields in `yarn.lock` is generated by newer version of yarn package manager